### ISSUE TITLE
failed to pull and unpack image after rpull

### ISSUE THREAD

**Issue by itsmurugappan** [containerd/stargz-snapshotter#187]
I tried out the stargz snapshotter on k3s on a rpi cluster. Here is my set up info -> https://github.com/itsmurugappan/stargz-snapshotter-k3s.

Everything works out great, even on the rpi cluster the pull performance is good.

I was thinking of doing a "ctr-remote rpull" on all the nodes before the pod is created. My assumption was it would improve the start up times even more. But when i tried, the rpull was succesful but when i created the pod i got the below error

```
Failed to pull image "ghcr.io/itsmurugappan/java-hw-estgz": rpc error: code = Unknown desc = failed to pull and unpack image "ghcr.io/itsmurugappan/java-hw-estgz:latest": failed to extract layer sha256:51159017383c06ece8aba377ce2382750e314a0f14cfbfe04ae4d11600a63b36: mount callback failed on /var/lib/containerd/tmpmounts/containerd-mount993838894: lchown /var/lib/containerd/tmpmounts/containerd-mount993838894/etc: operation not supported: unknown
```

```
"ghcr.io/itsmurugappan/go-hw-esgz:latest": rpc error: code = Unknown desc = failed to pull and unpack image "ghcr.io/itsmurugappan/go-hw-esgz:latest": failed to extract layer sha256:d845cd791b570f151d071dab99fdeada91d6eb3cf5e13398062a9eff64c13b7b: mount callback failed on /var/lib/containerd/tmpmounts/containerd-mount599162268: lchown /var/lib/containerd/tmpmounts/containerd-mount599162268/etc: operation not supported: unknown
```

Without doing rpull the pull and pod creation is successful.

**Comment by ktock** [comment]
Thanks for reporting this. Does adding `--namespace=k8s.io` option to `ctr-remote` fix this?
Thus `ctr-remote --namespace=k8s.io i rpull <your-image>`.

**Comment by itsmurugappan** [comment]
yes. it does. Thanks a lot for quick response.

**Comment by ktock** [comment]
@itsmurugappan This is fixed in v0.2.0 https://github.com/containerd/stargz-snapshotter/releases/tag/v0.2.0
